### PR TITLE
Doc/install/b-m/index: Add install overview video

### PR DESCRIPTION
**Pull request by joshix**
Upstream of the changes mocked up for staging in https://github.com/coreos-inc/coreos-pages/pull/1903

### CODE REVIEW

**Comment by joshix** [comment]
This video could at minimum use one more revision to eliminate the long black gaps before we finalize, sync, and publish.

**Comment by robszumski** [comment]
new video embed: https://youtu.be/61tSi50rfRI

File: Documentation/install/bare-metal/index.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ This guide walks through a bare metal installation of Tectonic utilizing PXE-bas
 
 For a more complete list of hardware and networking requirements, please see the bare metal [requirements document][requirements].
 
+## Overview video
+
+This overview video helps understand the big picture before beginning the deployment steps:
+
+<div class="tectonic-demo">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/A05T5n8taAQ" frameborder="0" allowfullscreen></iframe>
```
Comment (robszumski) [inline Documentation/install/bare-metal/index.md]: try putting just a width or a height

Comment (robszumski) [reply Documentation/install/bare-metal/index.md]: better yet, reference how the video the tectonic landing page works

**Comment by joshix** [comment]
produces https://a5ec0af4537abc346ec38faf3db5aa0de7106fdb.staging.pages.web.aws.coreos.com/tectonic/docs/latest/install/bare-metal/

**Pull request merged**